src/Extend/Mail.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Extend;
+
+use Flarum\Extension\Extension;
+use Illuminate\Contracts\Container\Container;
+
+class Mail implements ExtenderInterface
+{
+    protected $addDrivers = [];
+
+    public function add($humanName, $driver)
```

Comment:
Please add doc blocks explaining the parameters. The human name isn't really a human name (it is currently shown in the dropdown, but that could change) - it's more like an identifier. The driver must be the name of a class (or binding) implementing the `MailDriverInterface`.